diff --git a/data/src/emulator.js b/data/src/emulator.js
index a117c86..60f10f7 100644
--- a/data/src/emulator.js
+++ b/data/src/emulator.js
@@ -1238,6 +1238,8 @@ class EmulatorJS {
             this.paused = false;
             if (this.touch) {
                 this.virtualGamepad.style.display = "";
+            } else if (this.defaultCoreOpts && this.defaultCoreOpts.supportsMouse) {
+                this.canvas.classList.remove("ejs-canvas-no-pointer");
             }
             this.handleResize();
             if (this.config.fullscreenOnLoad) {
