--- emulator.js.orig	2026-09-14 17:56:11.528869975 -0600
+++ emulator.js	2026-09-14 17:56:11.547272492 -0600
@@ -1194,6 +1194,8 @@
             this.paused = false;
             if (this.touch) {
                 this.virtualGamepad.style.display = "";
+            } else if (this.defaultCoreOpts && this.defaultCoreOpts.supportsMouse) {
+                this.canvas.classList.remove("ejs-canvas-no-pointer");
             }
             this.handleResize();
             if (this.config.fullscreenOnLoad) {
