diff --git a/data/loader.js b/data/loader.js
index f9d76b3..f275004 100644
--- a/data/loader.js
+++ b/data/loader.js
@@ -243,6 +243,7 @@ async function prepareLanguage() {
         ["ready", window.EJS_ready],
         ["start", window.EJS_onGameStart],
         ["loadState", window.EJS_onLoadState],
+        ["quickLoadState", window.EJS_onQuickLoadState],
         ["saveState", window.EJS_onSaveState],
         ["loadSave", window.EJS_onLoadSave],
         ["saveSave", window.EJS_onSaveSave],
